Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/debian:buster-slim
+FROM docker.io/debian:bullseye-20210927-slim
```

Comment:
```suggestion
FROM docker.io/debian:11-slim
```

Reply:
I will use `11.0` as it is a more robust tag:) But thanks for the hint!